### PR TITLE
added ReCAPTCHA  service

### PR DESCRIPTION
**Pull request by Jehfkemsy**

### CODE REVIEW

File: src/services/captcha.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      if (data.success === undefined || !data.success) {
+        throw "Failed CAPTCHA verification";
+      }
+      return "CAPTCHA verified";
+    })
+    .catch(e => (isFailed = e)); // this is some hacky stuff... don't have the time to fix rn
+  if (isFailed) throw isFailed;
+};
+
+export default { validate };
```
Comment (Tommy2016x) [inline src/services/captcha.js:41]: are you using this anywhere else?

Comment (Jehfkemsy) [reply src/services/captcha.js:41]: you can if we add any other forms... you can also have different API keys for different forms

File: src/routes/index.js [hunk 2]
```diff
@@ -47,16 +54,18 @@ apiRouter.get("/admin/remind_confirm", adminAuthMiddleware, application.remindCo
 apiRouter.get("/admin/remind_apply", adminAuthMiddleware, application.remindApply);
 apiRouter.post("/admin/schedule/create", adminAuthMiddleware, schedule.create);
 apiRouter.get("/admin/schedule/read", adminAuthMiddleware, schedule.read);
-apiRouter.put("/admin/schedule/update", adminAuthMiddleware, schedule.update)
-apiRouter.delete("/admin/schedule/remove", adminAuthMiddleware, schedule.remove);
-
+apiRouter.put("/admin/schedule/update", adminAuthMiddleware, schedule.update);
+apiRouter.delete(
+  "/admin/schedule/remove",
```
Comment (Tommy2016x) [inline src/routes/index.js]: Also why did you change the format of these for only some of the routes?

Comment (Jehfkemsy) [reply src/routes/index.js]: why did I do? I can't tell what you are referring to?

**Review by Tommy2016x — APPROVED** [review]
